File: JavaScript/Iniciante/1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

linha = lines.shift().split(" ")
a = parseFloat(linha[0])
b = parseFloat(linha[1])
c = parseFloat(linha[2])
pi = 3.14159

triangulo = (a * c) / 2
circulo = pi * Math.pow(c, 2)
trapezio = (a + b) * c / 2
quadrado = b * b
retangulo = a * b

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`)
console.log(`CIRCULO: ${circulo.toFixed(3)}`)
console.log(`TRAPEZIO: ${trapezio.toFixed(3)}`)
console.log(`QUADRADO: ${quadrado.toFixed(3)}`)
console.log(`RETANGULO: ${retangulo.toFixed(3)}`)

File: C#/Iniciante/1012.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] valores = Console.ReadLine().Split(' ');

        double a = Convert.ToDouble(valores[0]);
        double b = Convert.ToDouble(valores[1]);
        double c = Convert.ToDouble(valores[2]);
        double pi = 3.14159;
        
        double triangulo = (a * c) / 2;
        double circulo = pi * (c * c);
        double trapezio = ((a + b) * c) / 2;
        double quadrado = b * b;
        double retangulo = a * b;
        
        Console.WriteLine($"TRIANGULO: {triangulo:F3}");
        Console.WriteLine($"CIRCULO: {circulo:F3}");
        Console.WriteLine($"TRAPEZIO: {trapezio:F3}");
        Console.WriteLine($"QUADRADO: {quadrado:F3}");
        Console.WriteLine($"RETANGULO: {retangulo:F3}");

    }

}
